core/crypto/crypto.odin

```diff
@@ -57,7 +57,7 @@ rand_bytes :: proc (dst: []byte) {
 }
 
 // has_rand_bytes returns true iff the target has support for accessing the
-// system entropty source.
+// system entropy source.
 has_rand_bytes :: proc () -> bool {
-	return _has_rand_bytes()
```

Comment:
What's the rationale from going to a constant to a call? Seems counter intuitive to me.

Reply:
"I added the call a short while ago, and felt bad about breaking compatibility."

I can get rid of the call entirely if that is better.